packages/ef-runtime-client/package.json

```diff
@@ -11,6 +11,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/css-modules": "^1.0.4",
```

Comment:
Can this be a dev dependency?

Reply:
Actually this should be removed. I was testing with CSS modules but it didn't work.